### DISCUSSION TITLE
SSH Remote Access Failed after newly installed TPot 20.06.2 port no.64295

### DISCUSSION THREAD

**Discussion by barrydekong** [telekom-security/tpotce#969] [Q&A]
Three days ago my TPot 20.06.2 on Cloud location was failed to connect with "Connection Timeout" by Putty through port no.64295 and I installed a new TPot at local VM for testing with the same issue! Anyone have the same issue or any solution resolve the issue?

**Comment by barrydekong** [comment]
Resolved after TPot update!

**Comment by Ganesh-403** [comment]
@barrydekong 
Since you mentioned the issue was resolved after updating T-Pot, it was likely caused by a bug or configuration issue that was addressed in a newer release.

For anyone encountering the same symptoms (`Connection timed out` on **port 64295**), I'd recommend checking the following before assuming it's an SSH problem:

1. **Verify the SSH service is running**

```bash
sudo systemctl status ssh
```

or

```bash
sudo systemctl status sshd
```

T-Pot changes the host's SSH daemon to **TCP port 64295** during installation, but it still uses the system's SSH service and OS users. :contentReference[oaicite:0]{index=0}

2. **Confirm SSH is listening on 64295**

```bash
ss -tlnp | grep 64295
```

or

```bash
netstat -tlnp | grep 64295
```

If nothing is listening, check `/etc/ssh/sshd_config` and verify:

```text
Port 64295
```

3. **Check whether the port is reachable**

From another machine:

```bash
nc -vz <tpot-ip> 64295
```

or

```bash
telnet <tpot-ip> 64295
```

If the connection times out, verify that:

- your cloud security group/firewall allows TCP/64295,
- there isn't an upstream firewall blocking the port,
- and `fail2ban` hasn't blocked your client IP after repeated failed login attempts. The T-Pot troubleshooting guide specifically mentions checking for fail2ban lockouts if you cannot log in. :contentReference[oaicite:1]{index=1}

4. **Review the SSH logs**

```bash
sudo journalctl -u ssh -xe
```

or

```bash
sudo journalctl -u sshd -xe
```

5. **If you're on an older release (e.g. 20.06.2)**

Since the original issue was resolved by updating, I'd strongly recommend upgrading to a supported T-Pot version. The project has had numerous improvements and fixes since the 20.06.x releases. :contentReference[oaicite:2]{index=2}

If the issue persists after upgrading, it would help to know:

- What is the output of:
  ```bash
  systemctl status ssh
  ss -tlnp | grep 64295
  sudo journalctl -u ssh -n 50
  ```
- Are you getting a **connection timeout**, **connection refused**, or an **authentication failure**? Those point to different root causes.

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
